.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "src/umpire/tpl/conduit"]
 	path = src/umpire/tpl/conduit
 	url = https://github.com/LLNL/conduit.git
+[submodule ".gitlab/radiuss-ci"]
```

Comment:
Could we put the submodule in the directory `.radiuss-ci` instead of `.gitlab/radiuss-ci` - it will get rid of the extra `gitlab` in the path, and also make it more obvious that the shared configurations can be used for more than just gitlab.

Reply:
OK. `.gitlab` directory will still exists right, though, confirm?

Reply:
I think the .gitlab directory can be removed now that we have all the compiler configurations in the radiuss-ci repo.

Reply:
.gitlab also contains the "ci" directory, with the pipeline description for LC machines.
These files will be partly generated in the near future (waiting for a new feature in Gitlab)

Reply:
So, as soon as they are partly generated, the remaining stuff could be reintegrated into .gitlab-ci.yml

Reply:
Except if you want to anticipate and have a large .gitlab-ci.yml in the meantime

Reply:
Ah, I understand. Let's keep the `.gitlab` directory, and move the RADIUSS submodule to `./.radiuss-ci`